src/read.jl

```diff
@@ -50,6 +56,15 @@ function read_case(filepath)
         CSV.File(joinpath(filepath, "gencost_after.csv"))
     )
 
+    # Set the PMAX for all profile-based generators to Inf; the true PMAX for profile-
+    # based generators will be determined by the provided profile
+    for g in unique(case["genfuel"])
+        gen_idx = case["genfuel"] .== g
+        if g in case["profile_resources"]
+            case["gen_pmax"][gen_idx] .= Inf
+        end
+    end
```

Comment:
Not sure whether it is more straightforward to do
```julia
for g in case["profile_resources"]
    case["gen_pmax"][case["genfuel"] .== g] .= Inf
end
```

Reply:
That should work. I'll make sure and implement it.

Reply:
Done.